### PR TITLE
Node.js: Improved Descriptions

### LINKED ISSUE #840: Node.js Suggestion: Improve Descriptions
### Description

This is a really popular Fathead, but it has a steady upwards trend for the Click Through Rate. This indicates that the content of the IA is not answering the users query. After a bit of looking about, I think we can improve this in 2 ways:

1) Improve the descriptions

screen shot 2017-02-27 at 6 58 15 pm

2) Add `node js` as a trigger word

## Get Started
- [x] 1) Claim this issue by commenting below
- [x] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-fathead/blob/master/CONTRIBUTING.md)
- [x] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [x] 4) Create a Pull Request
- [x] 5) Ping @bfmags for a review

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/node_js

### PR DESCRIPTION
**Pull request by Lord-Lavios**
## Description of new Instant Answer, or changes
Improves description for nodejs fathead.

## Related Issues and Discussions
Fixes #840 

## People to notify
@pjhampton @moollaza 

## Testing & Review
To be completed by Language Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)

**Code**
- [ ] Uniformly indented, well commented
- [ ] Fetch.sh and Parse.sh run without errors
- [ ] Output contains no blank lines, or multi-line entries
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/node_js

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by Lord-Lavios** [comment]
@pjhampton Will it automatically get re-deployed to beta with the new output.txt?

**Comment by moollaza** [comment]
@Lord-Lavios this is looking good 👍 

FYI please add "Fixes: " before the related issue you want to link. That way, when we merge the PR, GitHub will automatically close the IA that your description indicates this PR fixes.

e.g. `Fixes #123`

**Comment by moollaza** [comment]
@Lord-Lavios Thanks! This LGTM :+1:

File: lib/fathead/node_js/parse.js [hunk 1]
```diff
@@ -85,13 +84,23 @@ function getAbstract(obj) {
         stability = '<p  id="obj.stability">Stability: ' + obj.stability + ' - ' + obj.stabilityText + '</p>';
     }
 
-    a = '<section class="prog__container">' + a + code + stability + '</section>';
+    if(obj.events) {
+        events += `<span class = "prog_sub">Events</span><ul class="prog__ul">`;
```
Comment (moollaza) [inline lib/fathead/node_js/parse.js]: whoops -- this should be `prog__sub`

**Pull request merged**